src/App/Containers/ContainerBasePyImp.cpp

```diff
@@ -157,6 +157,10 @@ PyObject* ContainerBasePy::getObject(PyObject* args)
 PyObject* ContainerBasePy::hasObject(PyObject* args)
 {
     PyObject* obj = nullptr;
+    if (PyArg_ParseTuple(args, "O!", &(DocumentPy::Type), &obj))
```

Comment:
Is it the desired behaviour that if the argument is a Document it returns False? At least adding a comment would be good to make this clear.